src/components/shared/Alert.tsx

```diff
@@ -1,49 +1,53 @@
-import React from 'react';
-import Slide from '@material-ui/core/Slide';
-import Dialog from '@material-ui/core/Dialog';
-import DialogActions from '@material-ui/core/DialogActions';
-import DialogContent from '@material-ui/core/DialogContent';
-import DialogContentText from '@material-ui/core/DialogContentText';
-import DialogTitle from '@material-ui/core/DialogTitle';
-import Button from '@material-ui/core/Button';
+import React, { useEffect } from 'react';
 
-function Transition(props: any) {
-    return <Slide direction="up" {...props} />;
+import { Alert as MuIAlert, Color } from '@material-ui/lab';
+import CheckIcon from '@material-ui/icons/Check';
+import ErrorIcon from '@material-ui/icons/Error';
+import { makeStyles, Theme, createStyles } from '@material-ui/core/styles';
+
+export interface AlertProps {
+  alert: { message: string; status: string };
+  unMountAlert: () => void;
 }
 
-export default function Alert(
-    {
-        // handleClose,
-        // handleSubmit,
-        // isOpen,
-        // hasTwoButtons = false,
-        // submitButtonText,
-        // title,
-        // text,
+export const Alert = ({ alert, unMountAlert }: AlertProps) => {
+  const classes = useStyles();
+  const displayTimer = 2000;
+  console.log('alert', alert);
+
+  useEffect(() => {
+    setTimeout(unMountAlert, displayTimer); // unmount after 2 seconds
+  });
+  // only success has been implemented, since error is handled inside each form. But we'll pbly run into use cases for the error alert
+  const muiAlertProps: { icon: JSX.Element; severity: Color } =
+    alert.status === 'success'
+      ? { severity: 'success', icon: <CheckIcon fontSize="inherit" /> }
+      : { severity: 'error', icon: <ErrorIcon fontSize="inherit" /> };
+
+  return (
+    <MuIAlert
+      icon={muiAlertProps.icon}
+      severity={muiAlertProps.severity}
+      className={classes.root}
+      variant="filled"
+    >
+      {alert.message}
+    </MuIAlert>
+  );
+};
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    // transition animations?
+    root: {
+      position: 'fixed',
+      width: '95%',
+      height: 33,
+      top: 20,
+      left: 20,
+      right: 20,
+      zIndex: 100000, // too much?
+      padding: 2,
     },
```

Comment:
```suggestion
      padding: 2,
      transform: 'translateY(-100vh)',
      animation: `slide-in-from-top 0.5s forwards`,
    },
```

and then in /src/assets/global.css:
```css
@keyframes slide-in-from-top {
  from {
    transform: translateY(-100vh);
  }
  to {
    transform: translateY(0);
  }
}
```

Reply:
Can I add backwards to have it disappear too? Currently i have the component unmounting after 2 seconds, not sure how to run the unmount after an animation though

Reply:
btw I think i'll be disabling the css highlighting of the cells on hover. Gonna make it more google-sheet-esque to only highlight on select.